csi.proto

```diff
@@ -1224,6 +1224,17 @@ message NodeStageVolumeRequest {
   // This field is OPTIONAL and MUST match the volume_context of the
   // volume identified by `volume_id`.
   map<string, string> volume_context = 6;
+
+  // If SP has VOLUME_MOUNT_GROUP node capability and CO provides
+  // this field then SP MUST ensure that volume is mounted with
+  // provided volume_mount_group and all files and directories
+  // within the volume are readable and writable by the provided
+  // volume_mount_group.
+  // The value of volume_mount_group should be group_id or group name
```

Comment:
I think we have to be clear whether COs may specify a name or an ID here. If we can avoid specifically mentioning UNIX and Windows, and use language that applies equally well to both, that would be ideal.

For example, we could say it must be the ID of the group. On UNIX that would imply decimal-formatted GID string. On Windows that would imply a SID string. If we say that names are also acceptable, then I would expect that to be true on both Wndows and UNIX.

I want to be unambiguous what the CO is required to pass down here, and I want it to be unambiguous what the SP need to be prepared to deal with here.

Reply:
reworded as we discussed offline. Now this reads:

>   // The value of volume_mount_group should be group
  // identifier (as determined by underlying operating system)
  // which would be associated with workload that uses the volume.